src/env.rs

```diff
@@ -14,6 +14,11 @@ pub static BOT_TOKEN: OnceLock<String> = OnceLock::new();
 pub static LOG_LEVEL: OnceLock<tracing::Level> = OnceLock::new();
 
 pub fn init() {
+    dotenvy::dotenv().unwrap();
+
+    #[cfg(debug_assertions)]
```

Comment:
Is there a reason, debugging is turned on here?

Reply:
The cfg attribute checks if this config is turned on, which in this case is used to only include the .env.local file on debug (i.e. local) builds

Reply:
ah ok